tools/static-code-analysis/pmd/suppressions.properties

```diff
@@ -5,3 +5,6 @@ org.eclipse.smarthome.io.console.rfc147.internal.OSGiConsole=SystemPrintln
 org.eclipse.smarthome.core.internal.events.ThreadedEventHandler=EmptyIfStmt
 org.eclipse.smarthome.model.core.internal.ModelRepositoryImpl=AvoidCatchingNPE
 org.eclipse.smarthome.io.net.http.internal.SecureHttpClientFactory=AvoidThrowingRawExceptionTypes
+org.eclipse.smarthome.automation.core.internal.RuleRegistryImpl=CompareObjectsWithEquals
```

Comment:
Which line? I only found ` == null` and ` != null` in that file.

Reply:
The line is 513.